h2o-py/h2o/h2o.py

```diff
@@ -1257,6 +1259,102 @@ def make_metrics(predicted, actual, domain=None, distribution=None):
               data={"domain": domain, "distribution": distribution})
     return res["model_metrics"]
 
+gen_model_file_name = "h2o-genmodel.jar"
+h2o_predictor_class = "water.util.H2OPredictor"
+
+def find_file(name, path):
+    """
+    Helper function for predict_json() function to check if both MOJO and h2o-genmodel.jar are available
+    :param name: file name to find
+    :param path: folder/directory name with full path
+    :return: Successful path name or None if failed
+    """
+    for root, dirs, files in os.walk(path):
+        if name in files:
+            return os.path.join(root, name)
+
+def check_java_process_and_version():
```

Comment:
There is already a similar method in server.py (_find_java). This method is redundant.

Reply:
Using _find_java function instead it.